dashboard/config/routes.rb

```diff
@@ -12,12 +12,14 @@ def redirect_to_teacher_dashboard
 
   resources :user_levels, only: [:update]
 
-  resources :gallery_activities, path: '/gallery' do
+  resources :gallery_activities, only: [:index], path: '/gallery' do
```

Comment:
I believe it's unused, but I didn't remove it to be extra conservative with this change.